docker/start.py

```diff
@@ -60,25 +45,25 @@ def start_kms():
     for i in range(1, len(listen_ip)):
       command.append("-n")
       command.append(listen_ip[i] + "," + listen_port)
-
-  if enableSQLITE:
-    loggersrv.info("Storing database file to %s" % dbPath)
+  if want_webui:
     command.append('-s')
-    command.append(dbPath)
-    os.makedirs(os.path.dirname(dbPath), exist_ok=True)
+    command.append(db_path)
 
   loggersrv.debug("server_cmd: %s" % (" ".join(str(x) for x in command).strip()))
   pykms_process = subprocess.Popen(command)
+  pykms_webui_process = None
 
-  # In case SQLITE is defined: Start the web interface
-  if enableSQLITE:
-    time.sleep(5)  # The server may take a while to start
-    start_kms_client()
-    sqlite_cmd = ['sqlite_web', '-H', listen_ip[0], '--read-only', '-x',
-                  dbPath, '-p', sqlite_port]
-
-    loggersrv.debug("sqlite_cmd: %s" % (" ".join(str(x) for x in sqlite_cmd).strip()))
-    sqlite_process = subprocess.Popen(sqlite_cmd)
+  try:
+    if want_webui:
+      time.sleep(2) # Wait for the server to start up
+      pykms_webui_env = os.environ.copy()
+      pykms_webui_env['PYKMS_SQLITE_DB_PATH'] = db_path
+      pykms_webui_env['PORT'] = '8080'
+      pykms_webui_env['PYKMS_LICENSE_PATH'] = '/LICENSE'
+      pykms_webui_env['PYKMS_VERSION_PATH'] = '/VERSION'
+      pykms_webui_process = subprocess.Popen(['gunicorn', '--log-level', os.environ.get('LOGLEVEL'), 'pykms_WebUI:app'], env=pykms_webui_env)
```

Comment:
I don't like this use of subprocess, can we not import and dispatch gunicorn to a thread?